app/decorators/petition_search.rb

```diff
@@ -61,6 +65,15 @@ def execute_search_query
     end
   end
 
+  def execute_search_for_create_petition_query
+    Petition.search do |query|
```

Comment:
Could we possibly limit the number of results here to make the Solr return only the limited number of results (could be more efficient)?

```query.limit 3``` could work.

Maybe even fetch the actual limit from ```AppConfig``` so it would be easily changeable? :)

Also, I am not sure if we need to use ```query.facet :state``` because we don't need to group the results by the state.